### PR TITLE
CP-27247: Move tapdisk to a blkio cgroup

### PR DESCRIPTION
**Pull request by rosslagerwall**
This will be useful to prioritize general dom0 IO ahead of IO from VMs.

### CODE REVIEW

**Comment by rosslagerwall** [comment]
See CP-31088 and CP-31142 for details of the performance measurements of this change. To summarize:

    The latency of operations in dom0 is reduced when VMs are performing IO on the same disk.
    Performance of disk IO in general is not affected.

File: control/tap-ctl-spawn.c [hunk 1]
```diff
@@ -233,18 +233,23 @@ tap_ctl_spawn(const char *slice)
...
+	if (tap_ctl_move_to_cgroup(id, "blkio", "vm.slice") < 0)
+		EPRINTF("failed to move tapdisk %d to blkio cgroup slice: %s; ignoring.\n", id, strerror(errno));
 
 	/* Put the tapdisk in a cgroup slice (best-effort) */
 	if (!slice) {
 #ifndef TAP_CTL_NO_DEFAULT_CGROUP_SLICE
 		/* No option specified; move it to the default slice */
-		if (tap_ctl_move_to_cgroup(id, "/") < 0)
+		if (tap_ctl_move_to_cgroup(id, "cpu", "/") < 0)
 			EPRINTF("failed to move tapdisk %d to default cgroup slice: %s; ignoring.\n", id, strerror(errno));
```
Comment (MarkSymsCtx) [inline control/tap-ctl-spawn.c]: Should these contain 'cpu' in line with the one above having 'blkio'?

**Comment by MarkSymsCtx** [comment]
Please sign off the change.

File: control/tap-ctl-spawn.c (hunk 1 above)
Comment (rosslagerwall) [reply control/tap-ctl-spawn.c]: Fixed.

Comment (MarkSymsCtx) [reply control/tap-ctl-spawn.c]: LGTM

**Pull request merged**